File: C++/18.cpp
#include <iostream>

using namespace std;

int main() {
   
    int size;

    cout << "Enter the number of elements: ";
    cin >> size;

    int list[size];

    cout << "Enter elements of the lis: " << endl;

    for (int i = 0; i < size; i++)
        cin >> list[i];

    

    cout << "Sum of the elements of the list: " << list[0] + list[size-1]<<endl;
    cout << "Difference of the elements of the list: " << list[0] - list[size-1]<<endl;
    cout << "Product of the elements of the list: " << list[0] * list[size-1]<<endl;

}

File: Python/18.py
myList = []

numElements = int(input("How many elements would you like to add? "))

for i in range(numElements):
    element = int(input("Enter an element: "))
    myList.append(element)

print("Sum of the elements of the list: ", myList[0] + myList[numElements - 1])
print("Difference of the elements of the list: ", myList[0] - myList[numElements - 1])
print("Product of the elements of the list: ", myList[0] * myList[numElements - 1])
